scripts/pylib/twister/twisterlib/platform.py

```diff
@@ -50,41 +54,58 @@ def __init__(self):
         self.filter_data = dict()
         self.uart = ""
         self.resc = ""
+        self.qualifier = None
+
+    def load(self, board, target, aliases, data):
+        """Load the platform data from the board data and target data
+        board: the board object as per the zephyr build system
+        target: the target name of the board as per the zephyr build system
+        aliases: list of aliases for the target
+        data: the data from the twister.yaml file for the target
+        """
+        self.name = target
+        self.aliases = aliases
+
+        # Get data for various targets and use the main board data as a
```

Comment:
I suppose the related documentation update is planned soon, right ?
https://docs.zephyrproject.org/latest/develop/test/twister.html#board-configuration